api/libproject/devstart.rb

```diff
@@ -18,15 +19,20 @@ def initialize(command)
         self.project = project
       end
       parser.on("--account [ACCOUNT]",
-           "Account to use when creating credentials (your.name@pmi-ops.org)") do |account|
+           "Account to use when creating credentials (your.name@pmi-ops.org); "\
+           "use this or --creds_file") do |account|
         self.account = account
       end
+      parser.on("--creds_file [CREDS_FILE]",
+           "Path to a file containing credentials; use this or --account.") do |creds_file|
+        self.creds_file = creds_file
+      end
     end
   end
 
   def parse args
     self.parser.parse args
-    if self.project == nil || self.account == nil
+    if self.project == nil || !((self.account == nil) ^ (self.creds_file == nil))
```

Comment:
When using gsutil we first activate service account credentials (which we create if --account is specified, or use from the path in --creds_file if that's specified.)

So it should be OK.